### PR TITLE
User/vchintalapat/dotnet9 stack updates

### PR DESCRIPTION
**Pull request by Saipriya-1144**
DotNet 9 stackupdate for bookworm:
Created a Dockerfile for creating the runtime image for version 9
Created an sdk for DotNet 9

### CODE REVIEW

File: images/runtime/dotnetcore/9.0/bookworm.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# dotnet tools are currently available as part of SDK so we need to create them in an sdk image
+# and copy them to our final runtime image
+FROM mcr.microsoft.com/dotnet/sdk:9.0-preview AS tools-install
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-sos
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-trace
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-dump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-counters
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-gcdump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-monitor --version 8.*
```
Comment (CodingIsBliss) [inline images/runtime/dotnetcore/9.0/bookworm.Dockerfile:9]: version 9

**Review by CodingIsBliss — APPROVED** [review]

**Comment by CodingIsBliss** [comment]
lets add a few tests

File: images/runtime/dotnetcore/9.0/bookworm.Dockerfile (hunk 1 above)
Comment (sarsharma) [reply images/runtime/dotnetcore/9.0/bookworm.Dockerfile:9]: we have 9-preview available for dotnet monitor 
https://github.com/dotnet/dotnet-monitor/releases

**Review by sarsharma — APPROVED** [review]

File: tests/Oryx.RuntimeImage.Tests/Python/PythonImagesTests.cs [hunk 2]
```diff
@@ -18,7 +18,6 @@ public PythonImagesTest(ITestOutputHelper output) : base(output)
         {
         }
 
-        [SkippableTheory]
```
Comment (CodingIsBliss) [inline tests/Oryx.RuntimeImage.Tests/Python/PythonImagesTests.cs:21]: We do need to add Theory here

File: tests/Oryx.RuntimeImage.Tests/Python/PythonImagesTests.cs [hunk 3]
```diff
@@ -57,7 +56,6 @@ public void PythonBusterRuntimeImage_Contains_VersionAndCommit_Information(strin
                 result.GetDebugInfo());
         }
 
-        [SkippableTheory]
```
Comment (CodingIsBliss) [inline tests/Oryx.RuntimeImage.Tests/Python/PythonImagesTests.cs:60]: same

**Review by CodingIsBliss — COMMENTED** [review]
LGTM

**Review by CodingIsBliss — APPROVED** [review]

**Review by sarsharma — APPROVED** [review]

**Pull request merged**